src/Controls/samples/Controls.Sample.UITests/Issues/Issue20896.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage 
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    x:Class="Maui.Controls.Sample.Issues.Issue20896">
+    <ContentPage.Resources>
+        <ResourceDictionary>
+
+            <Style TargetType="Button">
+                <Setter Property="ImageSource" Value="groceries.png" />
+                <Setter Property="HorizontalOptions" Value="Center" />
+                <Setter Property="TextColor" Value="Black" />
+                <Setter Property="HeightRequest" Value="36" />
+            </Style>
+        
+        </ResourceDictionary>
+    </ContentPage.Resources>
+    <ScrollView>
+        <StackLayout
+            Padding="12">
+            <Label 
+                Text="This is a manual test to verify the use of ContentLayout in Catalyst. Remember, edit the info.plist to a UIDeviceFamily = 6. If the icon is drawn, the test has passed."/>
+            <Button
+                ContentLayout="Left, 24"
```

Comment:
With the changes, we are rendering now the Button image, but ContentLayout is not working with UIDeviceFamily 6. The ImageInsets and TitleInsets values are correct (and work on iOS, or Catalyst using another device family). Should we update the implementation and use `UIButtonConfiguration` with the `ImagePlacement` property?